backend/apis/parse-docket-pdf.utils.js

```diff
@@ -89,13 +89,80 @@ function parseCharges(lines, sections) {
 }
 
 function parseAccountSummary(lines, sections) {
-  const accounts = [];
+  let subsections = [];
+  let balances = [];
+  const sectionLastIndex = sections.filter(section => {
+    return section.name === "PROCEEDINGS" && section.lineNumber;
+  })[0].lineNumber;
+
+  function addAccounts(line, index) {
+    const words = line.split(/\s/);
+    const isSubsection =
+      words.length > 0 &&
+      words[0].toUpperCase() === words[0] &&
+      !words.includes("CASE") && // prevents CASE NUMBER lines from defining a subsection
+      words[1]; // prevents line new page diving line from defining a subsection;
```

Comment:
The problem was not about new line characters or spaces. What was happening was that the new page line separator:
_____________________________________________________________________________________________________

was passing the easy case and being assigned as the value of the "name" property. That is, (1) there was at least one word (words.length > 0) because words[0] === "____...", (2) words[0].toUpperCase() === words[0] ("_".toUpperCase() returns "_"), (3) words[0] !== "CASE". (I added this condition to prevent the CASE NUMBER line from starting a new subSection.) Adding that last condition (words[1]) returned the right answer. 
"_".trim() returns "_".

I noticed after that the comment itself is very unclear. So sorry about the confusion.

Reply:
Ah this line in the pdf?

https://github.com/JustUtahCoders/utahexpungements.org/blob/c9522469b27675264611ce65654147627a1566bd/backend/apis/docket-pdf-test-files/account-summary.js#L46

Yeah maybe we could just ensure there's at least one alphabetical character in the line then? That way if there is a section that is only one word, the parsing would still catch it.

```
```suggestion
     /[a-zA-Z]/.test(line)
```

Reply:
I updated it to this solution.